include/apps/watchfaces/OswAppWatchfaceMix.h

```diff
@@ -14,8 +14,11 @@ class OswAppWatchfaceMix : public OswApp {
   virtual void stop() override;
   ~OswAppWatchfaceMix(){};
 
+  void dateDisplay();
+  void digitalWatchDisplay();
```

Comment:
There is a big difference between what is used in Mix and what is used in Digital. Think of a way to separate it completely rather than reuse it, or i can call only the functions that `timeOutput`.